### PR TITLE
1 application show page

### LINKED ISSUE #1: Application Show Page (User Story 1)

### PR DESCRIPTION
**Pull request by AlecKap**
### Describe your changes
- Test: add tests for application show page.
- Feat: add application instance variable to show action.
- View: add html and erb to render application attributes.

### Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] I have added thorough tests - unit and integrated.

### CODE REVIEW

File: app/controllers/applications_controller.rb [hunk 1]
```diff
@@ -8,6 +8,7 @@ def create
   end
 
   def show
+    @application = Application.all.find(params[:id])
```
Comment (grace-joh) [inline app/controllers/applications_controller.rb]:
```suggestion
    @application = Application.find(params[:id])
```

Comment (grace-joh) [reply app/controllers/applications_controller.rb]: `.find` works without calling all as well!

File: spec/features/applications/application_show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    test_data
+  end
+  
+  describe 'As a visitor when I visit the application show page' do
+    it 'displays the applinaction attributes name, 
+      full address, description, name of all pets, and the status' do
+      
+      visit "/applications/#{@app1.id}"
+save_and_open_page
+      expect(page).to have_content('Name: Grace Joh')
```
Comment (grace-joh) [inline spec/features/applications/application_show_spec.rb:14]: How do you feel about interpolating `#{@app1.name}` to keep the test dynamic for if test data was to change?

**Review by grace-joh — CHANGES_REQUESTED** [review]
See comments!

File: spec/features/applications/application_show_spec.rb (hunk 2 above)
Comment (AlecKap) [reply spec/features/applications/application_show_spec.rb:14]: Lets follow up with an instructor about the standard convention here, but leave it for now.

Comment (grace-joh) [reply spec/features/applications/application_show_spec.rb:14]: Sounds good!

**Review by grace-joh — APPROVED** [review]
Comments resolved

**Pull request merged**